tools/topology/sof-byt-codec.m4

```diff
@@ -56,11 +56,11 @@ DAI_ADD(sof/pipe-dai-playback.m4,
 	1000, 1, 0, SCHEDULE_TIME_DOMAIN_DMA)
 
 # PCM Media Playback pipeline 3 on PCM 1 using max 2 channels of s32le.
-# 4000us deadline on core 0 with priority 0
+# 1000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(sof/pipe-pcm-media.m4,
-	3, 1, 2, s16le,
-	4000, 0, 0,
-	8000, 96000, 48000,
+	3, 1, 2, s32le,
+	1000, 0, 0,
+	8000, 48000, 48000,
```

Comment:
humm, how does removing 96kHz -> 48kHz reduce RAM consumption.  It's rather straightforward with a 1/2 ratio and subsampling?
Or did you mean that this included 88.2->48kHz which is indeed useless?
Wondering if the range is the problem, and if we need a discrete set of frequencies instead.

Reply:
@plbossart I guess because buffering is always 1ms so 96000 -> bigger buffers after src...

Reply:
before src I meant...

Reply:
Yep, the macro to compute buffer sizes handles this.